man/fi_mr.3.md

```diff
@@ -575,10 +575,14 @@ into calls as function parameters.
 ```c
 struct fi_mr_attr {
     union {
-      const struct iovec *mr_iov;
-      const struct fi_mr_dmabuf *dmabuf;
+        const struct iovec *mr_iov;
+        const struct fi_mr_dmabuf *dmabuf;
+        const struct fid_mr *base_mr;
     };
-    size_t             iov_count;
+    union {
+        size_t         iov_count;
+        size_t         length;
+    }
```

Comment:
That's a good question. I don't have preference either way. I do see one benefit of using iovec -- base_mr becomes a separate field and can be used to determine if a sub-MR is to be created instead of using a new flag.